src/app/backend/replicasetdetail.go

```diff
@@ -46,6 +46,14 @@ type ReplicaSetDetail struct {
 
 	// Detailed information about Pods belonging to this Replica Set.
 	Pods []ReplicaSetPod `json:"pods"`
+
+	// Internal endpoints of all Kubernetes services have the same label selector as this Replica Set.
+	// Endpoint is DNS name merged with ports.
+	InternalEndpoints []string `json:"internalEndpoints"`
```

Comment:
I think this model needs a submodel for the service. I.e.., replicaSetDetail contains service information. Replica list contain only endpoints, but that's all they're going to display. Here we have detailed information and label selector for the service. Later we'll add name and possibly other things. I think it is worth to create a nested or named struct here.

What do you think?